shared/ui-components/src/page_components/DashBlock.tsx

```diff
@@ -0,0 +1,42 @@
+import React from "react"
+import "./DashBlocks.scss"
+
+interface DashBlockProps {
+  href?: string
+  title: string
+  subtitle?: string
+  icon: JSX.Element
+  children?: JSX.Element | JSX.Element[]
```

Comment:
Let's avoid the use of `JSX.Element` per https://github.com/bloom-housing/bloom/issues/450